### ISSUE TITLE
More than 100 frame errors, UART RX was disabled

### ISSUE THREAD

**Issue by nextdayy** [LoRaTracker/GPSTutorial#2]
Hi there, I followed your tutorial after my GPS stopped working.

I recently coated my entire board in gel nail varnish to protect it from water, and did a freezer test with it for 3 hours. During this time, everything worked fine (except the GPS which I switched off because it wasn't going to be doing anything inside a freezer), but when I went to use the GPS again a couple days later, it was no longer getting a fix. 
So I tried printing out all the data as your tutorial suggests, and I got the following output:
```
$GNTXT,01,01,01,More than 100 frame errors, UART RX was disabled*70
$GNRMC,,V,,,,,,,,,,N*4D
$GNVTG,,,,,,,,,N*2E
$GNGGA,,,,,,0,00,99.99,,,,,,*56
$GNGSA,A,1,,,,,,,,,,,,,99.99,99.99,99.99*2E
$GNGSA,A,1,,,,,,,,,,,,,99.99,99.99,99.99*2E
$GPGSV,1,1,00*79
$GLGSV,1,1,00*65
$GNGLL,,,,,,V,N*7A
```

I've tried removing the varnish from around the antenna on the uBLOX-MAX-M8C Pico chip, but it still won't work. I'm using a RPi-Pico with Arduino-Pico and TinyGPS++, connected via Serial1 (hardware UART0)
So basically, you got any idea what that error means? Or is my chip just dead?

Thanks in advance for your help. :)
